AnkiDroid/src/test/java/com/ichi2/anki/dialogs/CreateDeckDialogTest.kt

```
@@ -30,6 +30,7 @@ import com.ichi2.anki.dialogs.CreateDeckDialog.DeckDialogType
 import com.ichi2.anki.dialogs.utils.input
 import com.ichi2.libanki.DeckId
 import com.ichi2.utils.positiveButton
+import okhttp3.internal.closeQuietly
```

is this correct? an internal import, but the method call below is on the activityScenario member

---

It's an extension to `closeable` which ignores exceptions

---

```kotlin
/** Closes this, ignoring any checked exceptions. */
fun Closeable.closeQuietly() {
  try {
    close()
  } catch (rethrown: RuntimeException) {
    throw rethrown
  } catch (_: Exception) {
  }
}
```